scenario/slice_source_pointer.yml

```diff
@@ -0,0 +1,38 @@
+input:
+    input.go: |
+        package slices_arrays
+
+        // goverter:converter
+        type Converter interface {
+            // goverter:convertSourcePointers
+            ConvertHouse(source DBHouseNames) APIHouseNames
+        }
+
+        type DBHouseNames struct {
+            Names *[]string
+        }
+
+        type APIHouseNames struct {
+            Names []string
+        }
+success: |
+    // Code generated by github.com/jmattheis/goverter, DO NOT EDIT.
+
+    package generated
+
+    import execution "github.com/jmattheis/goverter/execution"
+
+    type ConverterImpl struct{}
+
+    func (c *ConverterImpl) ConvertHouse(source execution.DBHouseNames) execution.APIHouseNames {
+    	var slices_arraysAPIHouseNames execution.APIHouseNames
+    	var stringList []string
+    	if (*source.Names) != nil {
```

Comment:
Additional parentheses there are looking redundant, but they are required for `(*source.Names)[i]` syntax. Looks like it is by-design.